src/js/components/aboutTheData/AgencyDetailsPage.jsx

```diff
@@ -0,0 +1,126 @@
+/**
+ * AgencyDetailsPage.jsx
+ */
+
+import React, { useEffect, useState } from 'react';
+import { useParams, Link } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { LoadingMessage, ErrorMessage } from 'data-transparency-ui';
+
+import { agencyPageMetaTags } from 'helpers/metaTagHelper';
+import { mockFetchAgencyOverview } from 'helpers/agencyV2Helper';
+
+import MetaTags from 'components/sharedComponents/metaTags/MetaTags';
+import Header from 'containers/shared/HeaderContainer';
+import Footer from 'containers/Footer';
+import StickyHeader from 'components/sharedComponents/stickyHeader/StickyHeader';
+import Note from 'components/sharedComponents/Note';
+import AgencyDetailsContainer from 'containers/aboutTheData/AgencyDetailsContainer';
+import { modalTitles, modalClassNames } from 'dataMapping/aboutTheData/modals';
+import BaseAgencyOverview from 'models/v2/agencyV2/BaseAgencyOverview';
+import ExternalLink from 'components/sharedComponents/ExternalLink';
+import AboutTheDataModal from './AboutTheDataModal';
+
+require('pages/aboutTheData/agencyDetailsPage.scss');
```

Comment:
FWIW there is a lot of duplicate styles in here.

Reply:
Agreed, we really should be able to combine it with the all Agencies page styles. It could become a mess with merge conflicts, but if you're mostly done with styling I can do that here!